File: python_semicolon/classwork/sentel.py
print("Welcome to The Mysterious Bank of Africa")
sentinel = "D"

alphabet = ""
balance = 0



while alphabet != sentinel:
	      
	alphabet =  input("Enter A to deposit B to withdrwal C to check balance and D to terminate\n")

	match (alphabet):
	

		case 'A':
			deposit = int (input("Enter the amount you want to deposit\n"))
			balance = balance + deposit

		case 'B':

			withdraw = int (input("Enter the amount you want to withdrwal\n"))
			if balance == 0 or withdraw > balance:
					print("insufficient funds")
			else:
				balance = balance  - withdraw

		case 'C':
		
			 if balance == 0:
					print("insufficient funds")

			 else:
		
                  	  		print("Your balance is ",  balance)

		
		


 	

File: python_semicolon/classwork/javacode/sentel.java
import java.util.Scanner;
public class Sentel{
	public static void main(String [] args){
System.out.println("Welcome to The Mysterious Bank of Africa");
Scanner sc= new Scanner(System.in);


 String sentinel = new String();
 double balance = 0;



while (!sentinel.equals("D")){

	System.out.println("Enter A to deposit and Enter B to withdrawal and C to check balance");
	      	   	
			sentinel = sc.next();
	switch (sentinel){
	

		case "A":
			 System.out.println("Enter the amount you want to deposit");
			 double deposit= sc.nextDouble();
			balance= balance + deposit;
				break;

		case "B":

			 System.out.println("Enter the amount you want to withdrawl");
			 double withdrawl= sc.nextDouble();
			 if(balance == 0 || withdrawl > balance){
				System.out.println("You cannot withdrawal");

	}
			else {
			      	balance= balance - withdrawl; 
}
				
				break;

		case "C":
				if(balance == 0)
					System.out.println("insufficient funds");
                  	  System.out.println("Your balance is " + balance);
				break;

		
		
}

 }


}

}